core-features/generate-client-and-cli-install-no-lockfile-npm/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eux
+
+VERSION=$(cat ../../.github/prisma-version.txt)
+
+cp -r ../_common/generate-client-and-cli-install/* .
+
+# we remove the lockfile to make sure that the install works without it
+# when no lockfile is found in a project, we will use npm for installs
+# currently, the root lockfile interferes with the detection, remove it
+rm ../../pnpm-lock.yaml
+rm package-lock.json
+
+# global instead instead
+npm install -g prisma@$VERSION
+prisma generate
```

Comment:
why global here?

Reply:
This is the original test, it tests that a global cli can install both prisma and client.

Reply:
Shouldn't the test have that in its name then?

Reply:
Done